src/app/gerente/gerente-routing.module.ts

```diff
@@ -0,0 +1,7 @@
+import { Routes } from '@angular/router';
+import { TelaInicioComponent } from './tela-inicio/tela-inicio.component';
+
+export const gerenteRoutes: Routes = [
```

Comment:
poderia ser com a primeira letra maiúscula também, se for seguir o padrão do prof, deixando GerenteRoutes, mas é bemmmm insignificante esse detalhe

Reply:
é estranho, na documentação do Angular eles usam minusculo `const routes: Routes = [
  { path: 'heroes', component: HeroesComponent }
];`